public/app/features/explore/RawListItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme } from '@grafana/data/src';
+import { useStyles } from '@grafana/ui/src';
+
+import { RawListValue } from './RawList';
+
+const getStyles = (theme: GrafanaTheme) => {
+  const reddish = theme.isDark ? '#ce9178' : '#a31515';
+  const greenish = theme.isDark ? '#73bf69' : '#56a64b';
+
+  return {
+    rowsWrapper: css`
+      width: 100%;
+    `,
+    rowWrapper: css`
+      border-bottom: 1px solid #ccc;
```

Comment:
If we had a theming color to replaces `#ccc` raw declaration. Would be nice to replace here too.

Reply:
These styles weren't in use, I removed the extras in this file